google/iam_compute_subnetwork.go

```diff
@@ -12,22 +12,22 @@ import (
 
 var IamComputeSubnetworkSchema = map[string]*schema.Schema{
 	"subnetwork": {
-		Deprecated: "This resource is in beta and will be removed from this provider. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/provider_versions.html for more details.",
+		Deprecated: "This field is in beta and will be removed from this provider. Use it in the the google-beta provider instead. See https://terraform.io/docs/providers/google/provider_versions.html for more details.",
```

Comment:
What's going on here?

Reply:
Looks like https://github.com/terraform-providers/terraform-provider-google/pull/2271 did this, and this was the final state of the file at HEAD in `2.0.0`. It was then removed (https://github.com/terraform-providers/terraform-provider-google/pull/2398). The "resource" text would be preferred here though.

Reply:
I'm agreed on the resource text being preferred here, but don't know how possible that is.